app/src/main/res/values/attrs.xml

```diff
@@ -27,6 +27,7 @@
     <attr name="toolbarDivider" format="reference"/>
     <attr name="menuCategoryText" format="reference"/>
     <attr name="preferenceSectionHeader" format="reference"/>
+    <attr name="menuItemButton" format="reference"/>
```

Comment:
I know we are severely lacking on the comments for the attrs, but I think this is a good opportunity to start adding descriptions for new attrs that we add. For instance, I couldn't tell where and what this attrs (color) would resolve to from the name itself.

```suggestion
    <!-- Menu item button color used in [DefaultToolbarMenu]. -->
    <attr name="menuItemButton" format="reference"/>
```